src/renderer/utils/platform.ts

```diff
@@ -9,3 +13,9 @@ export function isMacOS(): boolean {
 export function isWindows(): boolean {
   return process.platform === 'win32';
 }
+
+export const logDirectoryPaths = {
+  win32: path.join(process.env.APPDATA || '', Constants.APP_NAME),
```

Comment:
Curious if we can use something like https://github.com/megahertz/electron-log?tab=readme-ov-file#transport from `electron-log` to get the folder of the current log file 🤔